src/player.py

```
@@ -1,4 +1,5 @@
 from src.inventory import Inventory
+from src.ui import animate_travel, display_menu
```

Ruff (0.8.2)

`src.ui.display_menu` imported but unused

Remove unused import: `src.ui.display_menu`

(F401)